### ISSUE TITLE
[BUG] Quick sale and instant sale icons are not visible

### ISSUE THREAD

**Issue by 123455478** [IsThereAnyDeal/AugmentedSteam#1681]
### Expected Behavior
Click Sell Fast, Sell Instant to sell fast.

### Actual Behavior
Quick sale and instant sale icons are not visible

### Steps to Reproduce the Problem
not reproducible

### Specifications

  - Augmented Steam Version: 2.4.1
  - Browser Version: 
![1](https://github.com/IsThereAnyDeal/AugmentedSteam/assets/138566790/0fa36559-a532-40c3-801b-bd355c9fb216)

Google Chrome Version 114.0.5735.199 (Official Build) (64-bit)
![2](https://github.com/IsThereAnyDeal/AugmentedSteam/assets/138566790/c5d9d2bc-8165-4cf4-a465-76bb3bcf2a5e)

**Comment by candela97** [comment]
Can you check the browser console and see if you're getting timed-out (429 Too Many Requests) ?

**Comment by 123455478** [comment]
> Can you check the browser console and see if you're getting timed-out (429 Too Many Requests) ?

I didn't do too many refreshes or other tasks. After installing the augmented team, I went in and checked it, and I tried restarting the computer and steam log-out, but the status quo remained the same

Is there a solution?

**Comment by 123455478** [comment]
Okay. Should I delete it and reinstall it?

**Comment by candela97** [comment]
No, open the browser console (F12), and see if there're any errors when you select any marketable item.

**Comment by 123455478** [comment]
Please check if this is right
![1](https://github.com/IsThereAnyDeal/AugmentedSteam/assets/138566790/3b95854e-ae64-46a3-a539-44ec7e9b4de7)

**Comment by candela97** [comment]
That looks like the market home page, quick sell buttons are shown when you click on a marketable item in the inventory.

**Comment by 123455478** [comment]
But I can't see it Did I do something wrong?
![1](https://github.com/IsThereAnyDeal/AugmentedSteam/assets/138566790/ac718807-d731-4417-a9fc-a3d1da51ea77)

**Comment by candela97** [comment]
Oh! You're using the embedded inventory. Sadly the extension doesn't work on those, you'll have to visit your inventory page https://steamcommunity.com/my/inventory/

**Comment by 123455478** [comment]
OMG,,,, I didn't know I had to go into the inventory page. Thank you so much for all the answers. Do you have a sponsor channel? I want to sponsor it

**Comment by candela97** [comment]
No, we don't. But I believe ITAD gets commissions from some 3rd-party stores if you buy through links on ITAD (ITAD hosts this extension now since the original Enhanced Steam ceased development).

I'll mark this as a feature request to investigate whether we can get it working on embedded pages.

**Comment by 123455478** [comment]
All right. Thank you Have a good day
